packages/graphql-transformer-core/src/util/transfomConfigVersionMap.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Map of versions and there warning messages
+ */
+export const TRANSFORM_CONFIG_VERSION_MAP = {
+  3: directives => {
```

Comment:
what are the keys in the map out here? Would be helpful to describe it at the top of the file